etna/core/models/basepage.py

```diff
@@ -40,7 +40,7 @@
 
 @method_decorator(apply_default_vary_headers, name="serve")
 @method_decorator(apply_default_cache_control, name="serve")
-class BasePage(MetadataPageMixin, DataLayerMixin, Page, HeadlessPreviewMixin):
+class BasePage(MetadataPageMixin, DataLayerMixin, HeadlessMixin, Page):
```

Comment:
This breaks some tests but is the recommended way according to https://github.com/torchbox/wagtail-headless-preview?tab=readme-ov-file#usage

Reply:
`HeadlessPreviewMixin` works for now...